packages/hooks/src/__tests__/useFiatAccountList.spec.tsx

```diff
@@ -1,78 +1,138 @@
 import * as React from 'react';
-import { StoreProvider, mockStore } from '@deriv/stores';
 import { renderHook } from '@testing-library/react-hooks';
 import useFiatAccountList from '../useFiatAccountList';
+import { APIProvider, useFetch } from '@deriv/api';
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useFetch: jest.fn(),
+}));
+
+const mockUseFetch = useFetch as jest.MockedFunction<typeof useFetch<'authorize'>>;
 
 describe('useFiatAccountList', () => {
     test('should return an empty list if client has no account', async () => {
-        const mock = mockStore({});
+        mockUseFetch.mockReturnValue({
+            data: {
+                authorize: {
+                    account_list: [
+                        {
+                            account_category: 'wallet',
+                            account_type: 'doughflow',
+                            currency: 'USD',
+                            is_virtual: 1,
+                            loginid: 'CRW909900',
+                        },
+                    ],
+                    loginid: 'CRW909900',
+                },
+            },
+            isLoading: false,
+            isSuccess: true,
+        } as unknown as ReturnType<typeof mockUseFetch>);
```

Comment:
```suggestion
        });
```
@sergei-deriv Don't do type casting, This way we get rid of TS error, Instead better to use `@ts-expect-error` so later once the type is fixed that ignored error will raise error so we can come back and clean up.